commit/report.go

```diff
@@ -112,8 +114,30 @@ func (p *Plugin) Reports(
 func (p *Plugin) ShouldAcceptAttestedReport(
 	ctx context.Context, seqNr uint64, r ocr3types.ReportWithInfo[[]byte],
 ) (bool, error) {
-	if isActive, err := p.checkActiveInstance(ctx); err != nil || !isActive {
-		return false, err
+	// check if we support the dest, if not we can't do the checks needed.
+	supports, err := p.chainSupport.SupportsDestChain(p.oracleID)
```

Comment:
Do we want to make this check earlier in the flow? 
Why even create the report / run ocr sequence if we wouldn't accept it

Reply:
Yep, gonna rejig